_layouts/landing_page.html

```diff
@@ -111,7 +111,7 @@ <h1>Examples</h1>
                                 private, multi-party circuits (called “gamerooms”) and play tic tac toe with
                                 shared state.
                             </div>
-                            <a href ="/docs/{{ site.data.general.latest_version }}/examples/gameroom/" type="button" class="btn btn-primary">Learn More ➜</a>
+                            <a href ="/examples/#gameroom/" type="button" class="btn btn-primary">Learn More ➜</a>
```

Comment:
The trailing `/` should be removed for the anchor link to work properly.

Reply:
Fixed.